src/components/AppNavHeader.tsx

```diff
@@ -48,6 +88,7 @@ const defaultStyles = createStyles('AppNavHeader', (theme) => ({
     color: theme.colors.onSurfaceVariant,
   },
   backAction: {},
+  backActionIconAny: {},
```

Comment:
Not in this PR but I think we need to look at a way to inject props into a component. This kind of style doesn't easily convey to a consumer that they can set the `color` property to modify the component behavior.

Reply:
I agree.  This particular component is a bit tricky because it is passed itself as a prop into (and then rendered by) react navigation.  However, maybe an object of props could be passed into developerConfig and then accessed via the hook.